frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/speedsearch/DiscoverAction.java

```diff
@@ -1,27 +1,28 @@
-package com.virtuslab.gitmachete.frontend.actions.graphtable;
+package com.virtuslab.gitmachete.frontend.actions.speedsearch;
```

Comment:
Changes in this file are unrelated

Reply:
I would rather avoid piggybacking anything on bug fixing PRs (to lower the chances of providing another bug :rofl:)

---

Anyway... Why this change (rename)?
Where is the connection between this package and the IntelliJ speed search?

Afaik the speed search is an ability to simply write a text within some window/dialog and search (results marked with yellow on the light theme). If there is no connection to this feature please revert this change / rename to smth else to avoid confusion.

Reply:
Yeah, I don't like the `speedsearch` name either... just `unanchored` (?) would be better if this action doesn't belong to any particular `ActionPlace`

Reply:
TBH, speed search is probably only place where this action is available, so this is the reason of this change.
But ok, `unanchored` is also ok.

Reply:
CMIIW but I dare timidly suppose that you misunderstand the meaning of `speed search`. 

This is `Find Action...` dialog (?):
![image](https://user-images.githubusercontent.com/19799111/91550790-62476b00-e929-11ea-8c4d-2e7725ae5a70.png)

This is speed search functionality:
![image](https://user-images.githubusercontent.com/19799111/91551065-dd108600-e929-11ea-9abc-9829640627f1.png)

No action is "available from `speed search`".

Reply:
One can argue that speed search is also wired up to Ctrl+Shift+A... as in, IIUC it's "speed search" in the sense that it yields the results as you type (as opposed to how e.g. `grep` operates - only yielding the results once the search pattern is considered complete)